src/utility.ts

```diff
@@ -19,13 +19,20 @@ export const toHex = (buffer: Buffer): string => {
         .join("");
 };
 
+export const toArray = (hex: string): Uint8Array => {
+    return Uint8Array.from(Buffer.from(hex, "hex"));
+};
+
 /**
  * Gets account id from private key.
  * @param priv 32 byte hexadecimal string of private key
  * @returns 20 byte hexadecimal string of account id
  */
-export const getAccountIdFromPrivate = (priv: string): string => {
-    const publicKey = getPublicFromPrivate(priv);
+export const getAccountIdFromPrivate = async (
+    priv: string
+): Promise<string> => {
+    const crypto = await Crypto.init();
```

Comment:
It is weird to create Crypto class every function call, we have two choices:
1. Move this function into the `Crypto` class.
2. Parameterize the `Crypto` context.

Reply:
I'll choose 2. : )